### PR TITLE
fix(docs): update compatibility matrix fixing awscdk bucket info

### PR DESCRIPTION
**Pull request by marciocadev**
## Checklist

- [x] Title matches [Winglang's style guide](https://www.winglang.io/contributing/start-here/pull_requests#how-are-pull-request-titles-formatted)
- [ ] Description explains motivation and solution
- [ ] Tests added (always)
- [x] Docs updated (only required for features)
- [ ] Added `pr/e2e-full` label if this feature requires end-to-end testing

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://www.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

File: docs/docs/04-standard-library/compatibility/compatibility.json [hunk 1]
```diff
@@ -149,10 +156,17 @@
...
-      "aws-cdk": {"implemented": false}
+      "aws-cdk": {"implemented": true}
+    },
+    "getLocation": {
+      "sim": {"implemented": false},
+      "tf-aws": {"implemented": true},
+      "tf-azure": {"implemented": false},
+      "tf-gcp": {"implemented": false},
+      "aws-cdk": {"implemented": true}
     }
```
Comment (Chriscbr) [inline docs/docs/04-standard-library/compatibility/compatibility.json:170]: I believe this is a private method, no?

Comment (marciocadev) [reply docs/docs/04-standard-library/compatibility/compatibility.json:170]: Yes, sorry fixing now

File: libs/wingsdk/src/shared-aws/bucket.inflight.ts [hunk 2]
```diff
@@ -252,7 +253,7 @@ export class BucketClient implements IBucketClient {
    * @param key The key to reach
    * @param duration Time until expires
    */
-  public async signed_url(key: string, duration?: Duration): Promise<string> {
+  public async signedUrl(key: string, duration?: Duration): Promise<string> {
```
Comment (eladb) [inline libs/wingsdk/src/shared-aws/bucket.inflight.ts:256]: How come this didn't break any tests?

**Review by eladb — CHANGES_REQUESTED** [review]

File: libs/wingsdk/src/shared-aws/bucket.inflight.ts (hunk 2 above)
Comment (marciocadev) [reply libs/wingsdk/src/shared-aws/bucket.inflight.ts:256]: I went to check, this method has not been implemented, it throws an exception.

I will fix this in the next few days

**Pull request closed without merging**